### PR TITLE
Indexpage

### PR DESCRIPTION
**Pull request by asya-sou**
Merging basic style (asyastyle.css) & index page

### CODE REVIEW

**Comment by kisa-sou** [comment]
Do we have to involve @momo-sou in this?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+	<title> CSCI 39548 - Assignement 1 </title>title>
+	<link rel ="stylesheet" type="text/css" href="CSS/asyastyle.css">
+</head>
+
+<body>
+	<h1> Animals share their favorite animals, #s, colors </h1>
+	<div class="link">
+		<a href="asya.html"> Asya</a>
+		<a href="momo.html"> Momo </a>
```
Comment (momo-sou) [inline index.html]: if @kisa-sou link is above mine she may be happier about this project?..
```suggestion
<a href="kisa.html"> Kisa </a>
<a href="momo.html"> Momo </a>```

Comment (kisa-sou) [reply index.html]: It does look better this way

File: index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		<a href="kisa.html"> Kisa </a>
```
Comment (asya-sou) [inline index.html:16]: > if @kisa-sou link is above mine she may be happier about this project?..

```suggestion
 ```

**Review by asya-sou — COMMENTED** [review]
we have to remove duplicate link then

File: index.html (hunk 2 above)
Comment (kisa-sou) [reply index.html:17]: swap was enough, multiple line edits were suggested =___=

**Pull request merged**